src/main/java/uk/gov/register/configuration/RegisterContentPages.java

```diff
@@ -1,14 +1,17 @@
 package uk.gov.register.configuration;
 
+import java.util.List;
 import java.util.Optional;
 
 public class RegisterContentPages {
     private final Optional<String> registerHistoryUrl;
     private final Optional<String> custodianName;
+    private final Optional<List<String>> similarRegisters;
 
-    public RegisterContentPages(Optional<String> registerHistoryUrl, Optional<String> custodianName) {
+    public RegisterContentPages(Optional<String> registerHistoryUrl, Optional<String> custodianName, Optional<List<String>> similarRegisters) {
```

Comment:
Could you rename `RegisterContentPages.java` to something else? It was originally called "RegisterContentPages" because it contained links to the various content pages related to registers - i.e. the register history page, the spec and other registers guidance but it now contains things other than just "register content pages".